apps/hexagon_matmul/pipeline.cpp

```diff
@@ -80,6 +80,7 @@ class MatMul : public Generator<MatMul> {
                     .unroll(y);
 
                 AB.update(0)
+                    .prefetch(A, rk, 2)
```

Comment:
It doesn't actually matter, but I have a slight preference to move this after the reorder (i.e. swap this line and the line below).